.ibm/pipelines/utils.sh

```diff
@@ -404,13 +404,13 @@ check_operator_status() {
 
 # Installs the Crunchy Postgres Operator from Openshift Marketplace using predefined parameters
 install_crunchy_postgres_ocp_operator(){
-  install_subscription crunchy-postgres-operator openshift-operators v5 crunchy-postgres-operator certified-operators openshift-marketplace
+  install_subscription postgresql openshift-operators v5 postgresql community-operators openshift-marketplace
   check_operator_status 300 "openshift-operators" "Crunchy Postgres for Kubernetes" "Succeeded"
 }
 
 # Installs the Crunchy Postgres Operator from OperatorHub.io
 install_crunchy_postgres_k8s_operator(){
-  install_subscription crunchy-postgres-operator operators v5 postgresql operatorhubio-catalog olm
+  install_subscription postgresql openshift-operators v5 postgresql community-operators openshift-marketplace
```

Comment:
@albarbaro I haven't tested this on k8s. @zdrapela has more idea into how it's installed there.
I hope the following is correct:
```suggestion
  install_subscription postgresql operators v5 postgresql operatorhubio-catalog olm
```

Reply:
I've been experimenting with Crunchy on Kubernetes, but I had issues with it. The function `install_crunchy_postgres_k8s_operator` is currently commented out and K8s uses the integrated db.
The `openshift-marketplace` is not available on Kubernetes clusters, so `operatorhubio-catalog` is correct, but I need to check if the postgresql sill work fine.
Feel free to merge it as it is now. Enabling it on K8s will require some more work.

Reply:
I just reverted the changes. it should be good to go now